src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="group-distance">
+
+        <legend>Personal information</legend>
+
+        <div class="distance">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="distance">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="distance">
+          <label for="age">
+            How old are You?
+            <input
+                type="number"
+                name="age"
+                id="age"
+                min="1"
+                max="100"
+                value="12"
+            >
+          </label>
+        </div>
+
+        <div class="distance">
+          <label for="dataOfBirth">
+            Full date of birth:
+            <input
+              type="date"
+              name="dataOfBirth"
```

Comment:
don't write value for attributes in cameCase, use `-` instead
```suggestion
              name="data-of-birth"
```